test/e2e/kubevirt.go

```diff
@@ -1548,9 +1548,9 @@ fi
 			cudn       *udnv1.ClusterUserDefinedNetwork
 			vm         *kubevirtv1.VirtualMachine
 			vmi        *kubevirtv1.VirtualMachineInstance
-			cidrIPv4   = "10.128.0.0/24"
+			cidrIPv4   = "172.31.0.0/24" // subnet in private range 172.16.0.0/12 (rfc1918)
```

Comment:
hmm isn't this service cidr default value in ocp.. is this tested its working d/s? i think its gonna conflict with cluster ip range?

Reply:
Thank you for your time reviewing this :bow: